### DISCUSSION TITLE
Image automation silently failing

### DISCUSSION THREAD

**Discussion by steveww** [fluxcd/flux2#1694] [Q&A]
I followed the example workflow in the docs to set up image automation. All steps completed without error.

I modified my application source, built and pushed a new image 1.0.0 -> 1.0.1

Nothing happened! I expected Flux to see the new image version and deploy it.

flux get image policy - resolves to 1.0.1 but no deployment update????

What went wrong? How do I debug this?

**Comment by stefanprodan** [comment]
Have you defined an `ImageUpdateAutomation` object? Please post here `flux get image update`.

**Comment by steveww** [comment]
Found it!

flux create image update foo
--git-repo-ref always equals flux-system not the name of the real git repo. Confusing.

There should be a flux status command that checks all the referential integrity and connections to Github etc. It would make it much easier to debug when this don't work as expected.

Thanks for your offer of help.

**Reply by stefanprodan** [reply]
there is a `flux get all` command that tells you what's going on, there is also `flux logs`, `flux get sources all --all-namespace` and many other debug commands. Take a look at the docs please: https://fluxcd.io/docs/
